src/shared/utils.js

```diff
@@ -1,3 +1,14 @@
+
+/**
+ * Convenience wrapper to shorten the hasOwnProperty call
+ *
+ * @param {Object|Array{*}} object
+ * @param {String} property
+ */
+export function hasOwn (object, property) {
+  return Object.prototype.hasOwnProperty.call(object, property)
```

Comment:
i think you forgot to move to `hasOwn` here

Reply:
This is the `hasOwn` function definition? If I replace it here it'll just be a recursive no-op.

Reply:
ups. Sorry.